src/org/opensolaris/opengrok/analysis/c/CXref.lex

```diff
@@ -80,7 +78,12 @@ Number = (0[xX][0-9a-fA-F]+|[0-9]+\.[0-9]+|[1-9][0-9]*)(([eE][+-]?[0-9]+)?[ufdlU
 
 {Identifier} {
     String id = yytext();
-    writeSymbol(id, Consts.kwd, yyline);
+    // For historical reasons, CXref does not link Identifiers of length=1
```

Comment:
the reason was that lucene wasn't that performant back then (and it made no sense to index 1 char identifiers), we can lift this now, since lucene should be able to handle 1 char identifiers now ...